src/frontend/src/views/Data/components/SamplesView/components/SampleTableModalManager/components/DownloadModal/index.tsx

```diff
@@ -39,6 +41,9 @@ const DownloadModal = ({
   const [isMetadataSelected, setMetadataSelected] = useState<boolean>(false);
   const [isGisaidSelected, setGisaidSelected] = useState<boolean>(false);
   const [isGenbankSelected, setGenbankSelected] = useState<boolean>(false);
+  const state = store.getState();
+  const pathogen = selectCurrentPathogen(state);
+  const isGisaidTemplateEnabled = pathogen === "SC2" ? true : false;
```

Comment:
Prefer to use our enum, in case we ever change the slug, and to keep types tidy :)

```suggestion
  const isGisaidTemplateEnabled = pathogen ===Pathogen.COVID;
```

Reply:
awesome did not know we had this!